src/content/refi/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Regenerative finance (ReFi)
+description: An overview of current use cases of blockchain technology for environmental and social impact.
+lang: en
+template: use-cases
+emoji: ":recycle:"
+sidebarDepth: 2
+image: ../../assets/impact_transparent.png
+alt: ""
+summaryPoint1: An alternative application of cryptoeconomic systems focused on sustainability and addressing societal problems.
+summaryPoint2: A tool to bring unprecedented transparency and drastically scale up current environmental asset markets like the voluntary carbon market.
+summaryPoint3: Attempts to mitigate and/or reduce the impact of global and regional crises.
+---
+
+## The Roots of ReFi: Regenerative Capitalism and the Cypherpunks {#roots-of-refi}
+
+In his [2015 paper](https://capitalinstitute.org/wp-content/uploads/2015/04/2015-Regenerative-Capitalism-4-20-15-final.pdf) Regenerative Capitalism: How Universal Principles and Patterns Will Shape Our New Economy, John Fullerton of the Capital Institute set out an evolved version of capitalism intended to account for and address negative externalities such as climate change and entrenched inequality.
+
+Reflecting on the financial crisis of 2008, Fullerton joined many in questioning the long-term viability of early 21st century capitalism and sought to imagine “a self-organizing, naturally self-maintaining, highly adaptive Regenerative form of capitalism that produces lasting social and economic vitality for global civilization as a whole”.
+
+Drawing on the theory of [biomimicry](https://biomimicry.org/what-is-biomimicry/) and applying systems thinking to economics, Fullerton identified eight interconnected principles that would underlie systemic health:

```

Comment:
This section kind of leads me to expect a list of the eight interconnected principles. Suggest either removing the colon and turning the words "eight interconnected principles" into a link to an external resource that lists the principles, or listing them as bullets after the colon.

Reply:
We added a missing image - it's got KlimaDAO branding, if that's an issue maybe an in-house designer could adapt it?

Reply:
Also worth noting that any English text in images will not be translated. Given we have the image below but only in English, it could be helpful to still type these eight principles out, so when viewed in other languages that text will be properly translated.